payjoin-ffi/python/README.md

```diff
@@ -8,46 +8,51 @@ Grab the latest release with a simple:
 
 ```shell
 pip install payjoin
+
+# Or, for uv:
+uv add payjoin
 ```
 
 ## Running Tests
 
 Follow these steps to clone the repository and run the tests.
 
-
 ```shell
-git clone https://github.com/payjoin/rust-payjoin.git
-cd rust-payjoin/payjoin-ffi/python
+# Ensure you have uv installed:
+# https://docs.astral.sh/uv/getting-started/installation/
```

Comment:
There are, of course, other deps required (`gcc` compiler, for ex.) but these will be abstracted away in `uv2nix` in the next PR